docs/README.md

```diff
@@ -11,3 +13,4 @@ To build the documentation as HTML:
     cd docs
     make html
 
+To contribute to this repository, please follow these [guidelines](https://github.com/Microsoft/Recommenders/wiki/Coding-Guidelines). See also the [Sphinx documentation](https://sublime-and-sphinx-guide.readthedocs.io/en/latest/index.html) for the syntax of docstrings.
```

Comment:
Is the second link reference about docstrings? I clicked and it seems like reference about the entire sphinx.

Reply:
Probably this link is better https://www.sphinx-doc.org/en/master/usage/restructuredtext/index.html, it is just the rst reference.